src/components/AddBook.js

```diff
@@ -30,30 +30,30 @@ const AddBook = () => {
   };
   return (
     <>
-      <h2>Add new book</h2>
+      <h2 className={styles.formHeader}>Add new book</h2>
       <div className={styles.input}>
-        <form onSubmit={addBookHandler}>
+        <form className={styles.bookForm} onSubmit={addBookHandler}>
           <label htmlFor="author">
-            Author
             <input
               id="author"
               name="author"
               type="text"
               ref={authorInputRef}
-              placeholder="Author..."
+              className={styles.authorInput}
+              placeholder="Author"
             />
           </label>
           <label htmlFor="title">
-            Title
             <input
               id="title"
               name="title"
               type="text"
               ref={titleInputRef}
-              placeholder="Title..."
+              className={styles.titleInput}
+              placeholder="Title"
             />
           </label>
-          <button type="submit">Add Book</button>
+          <button type="submit" className={styles.formBtn}>Add Book</button>
         </form>
```

Comment:
### OPTIONAL 

- [ ] Kindly align the form to have same width with the booklist container 👍🏿 

 Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Screenshot_141](https://user-images.githubusercontent.com/69638013/167168723-147e6ddc-0e0d-4fa3-8f35-1275316a8399.png)  |  ![Screenshot_140](https://user-images.githubusercontent.com/69638013/167168688-adfd183c-1fe3-4357-b6d9-a2dc442e92fb.png)